test/e2e/framework/framework.go

```diff
@@ -54,6 +55,8 @@ import (
 const (
 	// DefaultNamespaceDeletionTimeout is timeout duration for waiting for a namespace deletion.
 	DefaultNamespaceDeletionTimeout = 5 * time.Minute
+	serviceAccountName              = "default"
+	SecretName                      = "registry-cred"
```

Comment:
Does this constant have to be exported? It doesn't seem so.

Can you add documentation for both new constants?

Alternatively, move them into the function which needs them (but again, with a comment).

Reply:
This constant is being used in one of the node conformance tests - https://github.com/kubernetes/kubernetes/pull/114625/files#diff-33732320a70e5099d257cef88bfc947d3b37a75d720e25935b78d58dd22af242R314

I'll add the comments

Reply:
`framework.SecretName` is not a good name: it doesn't describe the kind of secret that is being named here.

Perhaps renaming it and proper documentation can convince me that this deserves to be part of the API.